packages/trader/src/AppV2/Components/TradeParameters/Duration/container.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import { ActionSheet, CaptionText, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { useTraderStore } from 'Stores/useTraderStores';
+import DurationEndTimePicker from './datepicker';
+import { observer } from '@deriv/stores';
+import DurationChips from './chips';
+import DurationWheelPicker from './wheelpicker';
+
+const DurationActionSheetContainer = observer(
+    ({ selected_hour, setSelectedHour }: { selected_hour: number[]; setSelectedHour: (arg: number[]) => void }) => {
+        const { duration, duration_unit, duration_units_list, onChangeMultiple, expiry_time, contract_type } =
+            useTraderStore();
+        const [unit, setUnit] = useState(expiry_time ? 'et' : duration_unit);
+        const [selected_time, setSelectedTime] = useState([duration]);
```

Comment:
whys it an array

Reply:
It is because we have to pass it as an array inside array in wheelpicker